mirror-registry/README.md

```diff
@@ -114,24 +119,25 @@ apiVersion: mirror.openshift.io/v1alpha2
 kind: ImageSetConfiguration
 storageConfig:
   registry:
-    imageURL: mavazque-virt.cloud.lab.eng.bos.redhat.com:8443/ocp4/openshift4 # Registry where we will mirror the images
+    imageURL: registry.example.com:8443/ocp4/metadata:latest  # Registry where we will mirror the images
     skipTLS: true
 mirror:
-  ocp: # OCP Releases we want to mirror
-    channels:
-      - name: stable-4.9
-        minVersion: 4.9.21
-        maxVersion: 4.9.21
+  platform:
+    channels: # OCP Versions to mirror
+      - name: stable-4.10
+        minVersion: 4.10.10
+        maxVersion: 4.10.20
+    graph: true
   operators: # Operators we want to mirror
-    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9 # References entire catalog
-      headsOnly: false # References latest version of each operator in catalog (true is the default value and can be omitted)
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.10
+      full: true
       packages:
         - name: local-storage-operator
-          channels:
-            - name: 'stable'
         - name: advanced-cluster-management
-          channels:
-            - name: "release-2.4"
+        - name: openshift-gitops-operator
+        - name: ocs-operator
+        - name: odf-operator
```

Comment:
i dont think you need both ocs-operator and odf-operator

Reply:
Removed odf.